api/application.go

```diff
@@ -677,9 +801,14 @@ func (r *Application) With(m *model.Application) {
 			r.Identities,
 			ref)
 	}
-	for _, tag := range m.Tags {
-		ref := Ref{}
-		ref.With(tag.ID, tag.Name)
+}
+
+//
+// WithTags updates the resource with the associated tags.
+func (r *Application) WithTags(tags []model.ApplicationTag) {
```

Comment:
Curious why this is separate instead of:
```
With(m *model.Application, tags []model.ApplicationTag)
```

Reply:
Couldn't decide what felt cleaner. Happy to make it one method like that.